src/main/java/org/junit/ComparisonFailure.java

```diff
@@ -16,15 +16,15 @@
      * @see ComparisonCompactor
      */
     private static final int MAX_CONTEXT_LENGTH = 20;
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 2L;
 
     /*
      * We have to use the f prefix until the next major release to ensure
      * serialization compatibility. 
      * See https://github.com/junit-team/junit/issues/976
      */
```

Comment:
@marcphilipp Yes. I have removed this comment and all the others that reference the same issue. I have also found another class with an f-prefixed field name and fixed it. Please take a look.